### DISCUSSION TITLE
大家记得保存

### DISCUSSION THREAD

**Discussion by mikumifa** [mikumifa/biliTickerBuy#194] [公告]
BW结束就删

**Comment by WittF** [comment]
🫶

**Comment by 0Miku0831** [comment]
/(ㄒoㄒ)/~~

**Comment by NCsagiri** [comment]
大佬别走QAQ~

**Comment by XINGYI0309** [comment]
今天教朋友怎么用，教到我红温了,我感觉大佬做的引导已经足够好了啊（QWQ）

**Reply by bilibiliSilentH** [reply]
电脑小白是这样的。

**Reply by mikumifa** [reply]
其实主功能老早之前就有了，最新的更新全在做交互

**Reply by XINGYI0309** [reply]
还有人在我说过这是电脑用的之后还在问我，为什么手机用不了（）

**Reply by bilibiliSilentH** [reply]
最近真的是困难重重啊，致敬了。

**Reply by XINGYI0309** [reply]
我是上一年bw下载的好像，当时好像是因为elgedriver的问题，导致没法运行，只好自己手抢

**Reply by mikumifa** [reply]
现在有代替方式，不用webdriver也行，而且现在也确定了

win11专业版会自动带驱动

**Reply by XINGYI0309** [reply]
不过大部分人的都是家庭版（我刚看了我自己的也都还是家庭版）

**Comment by bilibiliSilentH** [comment]
webdriver要去下载驱动的。
&nbsp;


该邮件从移动设备发送


&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年6月28日(星期五) 晚上11:15
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mikumifa/biliTickerBuy] 大家记得保存 (Discussion #194)














 
我是上一年bw下载的好像，当时好像是因为elgedriver的问题，导致没法运行，只好自己手抢

**Reply by XINGYI0309** [reply]
是这样的，只是当时刚拿到电脑没两天，也是刚高中毕业，脱离没有电脑摸的苦海，真的不怎么明白，只以为是脚本出问题了（笑）

**Comment by Colin-Chen630** [comment]
🫶

**Comment by YourComputer011** [comment]
🫶

**Comment by Kizuna-AI0630** [comment]
感谢大佬开源🙇‍

**Comment by cuidongshan2024** [comment]
感谢大佬，感恩的心感谢有你
